lib/mdl.rb

```diff
@@ -106,6 +106,7 @@ def self.run(argv=ARGV)
     end
 
     if Config[:json]
+      require 'json'
```

Comment:
Can you remove the `require 'json'` on line 91? It was supposed to be here, not up there.